src/github.com/getlantern/flashlight/flashlight.go

```diff
@@ -217,6 +217,13 @@ func runClientProxy(cfg *config.Config) {
 	settings.Configure(cfg, version, buildDate)
 	proxiedsites.Configure(cfg.ProxiedSites)
 
+	httpClient, er := util.HTTPClient(cfg.CloudConfigCA, cfg.Addr)
```

Comment:
It depends.  Do you think there's a realistic chance chance that Go defaults will miss the CA of a Google-run service? :)